packages/core/src/types.ts

```diff
@@ -1700,14 +1700,15 @@ export interface StateConfig<
   tags?: Set<string>;
   machine?: StateMachine<TContext, TEvent, any, any, any, any, any, any, any>;
   _internalQueue?: Array<TEvent>;
+  timers?: ScheduledEvent[];
```

Comment:
should this be an array? this is not quite an ordered collection (order isn't that relevant to us) and when removing an item (when canceling an event) it would be easier to remove a key from an object than to remove an array item